### DISCUSSION TITLE
Goto/Label Implementation & Insensitive String Comparision?

### DISCUSSION THREAD

**Discussion by Mik39907** [ThioJoe/YT-Spammer-Purge#487] [Ideas]
Implementing a Goto in case something is written incorrectly or missed something that should be before any changes were made to the list of comments/accounts?
And also shouldnt the words be case insensitive?
https://stackoverflow.com/questions/319426/how-do-i-do-a-case-insensitive-string-comparison
https://stackoverflow.com/questions/438844/is-there-a-label-goto-in-python

**Comment by SeqBra** [comment]
What are you asking?

> And also shouldnt the words be case insensitive?

Yes, probably

**Comment by SeqBra** [comment]
I can't guarantee it, but at a glance of the search results of `lower` and `upper`, it looks like at least some case insensitivity is implemented 👍
